doc/modules/classes.rst

```diff
@@ -1003,6 +1003,28 @@ See the :ref:`metrics` section of the user guide for further details.
    metrics.pairwise_distances_chunked
 
 
+Plotting metrics
+----------------
+
+.. automodule:: sklearn.metrics.plot
```

Comment:
So we've decide to use sklearn.XXX.plot.plot_XXX, right?
We'll put things like ``plot_decision_boundary`` in ``sklearn.inspection``, right?